tests/e2e/specs/reception/import_samples_from_sequencescapeKinnexTubes.cy.js

```diff
@@ -0,0 +1,152 @@
+import SequencescapeKinnexTubeFactory from '../../../factories/SequencescapeKinnexTubeFactory.js'
+import PrinterFactory from '../../../factories/PrinterFactory.js'
+
+describe('Import samples from Sequencescape Kinnex Tubes', () => {
+  beforeEach(() => {
+    cy.intercept('v1/library_types?fields[library_types]=name,pipeline', {
+      fixture: 'tractionLibraryTypes.json',
+    })
+
+    cy.wrap(PrinterFactory()).as('printerFactory')
+    cy.get('@printerFactory').then((printerFactory) => {
+      cy.intercept('GET', '/v1/printers', {
+        statusCode: 200,
+        body: printerFactory.content,
+      })
+    })
+  })
+  describe('Successfully - V2', () => {
```

Comment:
What does rhe V2 mean?

Reply:
I think it perhaps refers to SS V2. I will change it to 'Importing samples from Sequencescape Kinnex Tubes'